File: app/src/main/java/com/marielm/android101/Repository.java
package com.marielm.android101;

import com.google.gson.annotations.SerializedName;

public class Repository {
  @SerializedName("name")
  public String name;

  @SerializedName("description")
  public String description;

  @SerializedName("updated_at")
  public String updatedOn;

  @SerializedName("id")
  public long id;

  @SerializedName("watchers_count")
  public int watchers;
}

File: app/src/main/java/com/marielm/android101/Repository.kt
package com.marielm.android101

import com.google.gson.annotations.SerializedName

data class Repository(var name: String? = null,
                      var description: String? = null,
                      @SerializedName("updated_at")
                      var updatedOn: String? = null,
                      var id: Long = 0,
                      @SerializedName("watchers_count")
                      var watchers: Int = 0
)
